roles/common/defaults/ssh.yaml

```diff
@@ -0,0 +1,38 @@
+---
+#server
+- name: configure SSH (daemon config)
+  hosts: Linux_Ubuntu_Server_1 #192.168.0.189
+  become: yes
+
+  tasks:
+    - name: replace Ports in /etc/ssh/sshd_config
+      lineinfile:
+        path: /etc/ssh/sshd_config
+        regexp: "^(.*)#Port(.*)$"
+        line: "Port 22"
+        backup: yes
+
+    - name: add Port in /etc/ssh/sshd_config
```

Comment:
Repetitive parts in some cases are best done through [loop](https://docs.ansible.com/ansible/latest/user_guide/playbooks_loops.html#iterating-over-a-simple-list)